src/traces/scattergl/convert.js

```diff
@@ -216,7 +216,7 @@ function convertTextStyle(gd, trace) {
         optsOut.font = {
             size: tfs * plotGlPixelRatio,
             family: tff,
-            weight: tfw,
+            weight: weightFallBack(tfw),
```

Comment:
Why is this required if we are already disallowing numeric weight values at the schema level?

Reply:
Good question. 
This is for `scattergl` with limited support of `weight` noting that for `arrayOk` attributes we do not check every element at the supply defaults step which could result in performance issues during interactions with the plot.
As a result this fall back is needed to map unsupported values in arrays to supported options.

Reply:
gotcha, thank you. Could you perhaps add that as a comment above the function definition?

Reply:
Addressed in [48d057f](https://github.com/plotly/plotly.js/pull/6990/commits/48d057fad97215e6c3d3a9924741918a76cd84ee).